internal/controller/ols_app_server_assets.go

```diff
@@ -258,7 +258,6 @@ func (r *OLSConfigReconciler) generateOLSConfigMap(ctx context.Context, cr *olsv
 			TranscriptsDisabled: cr.Spec.OLSConfig.UserDataCollection.TranscriptsDisabled || !dataCollectorEnabled,
 			TranscriptsStorage:  "/app-root/ols-user-data/transcripts",
 		},
-		IntrospectionEnabled: cr.Spec.OLSConfig.IntrospectionEnabled,
```

Comment:
@syedriko We want the flag for the TechPreview release right ? we enable the MC config only if this flag is on .

Reply:
We only want to merge this PR after https://github.com/openshift/lightspeed-operator/pull/739 merges. If that OLS merge makes it to TechPreveiw, so will this PR.

Reply:
True - but we want the behaviour (1) Default install WILL not have cluster_introspection (2) Admin wants to enable cluster introspection using a flag , which is cr.Spec.OLSConfig.IntrospectionEnabled